Sources/MessagingPush/PushContent.swift

```diff
@@ -0,0 +1,63 @@
+import Foundation
+#if canImport(UserNotifications)
+import UserNotifications
+
+/**
+ The content of a push notification. Single source of truth for getting and setting properties of a push notification.
+
+ Meant to be used by the SDK to set/get the push content but also designed for SDK user to further modify the push
+ if they wish.
+ */
+public class PushContent {
+    public var title: String {
+        didSet {
+            modifyNotificationContent()
```

Comment:
why do this over setting the individual pieces in each of these?

Reply:
I don't understand the question

Reply:
like
```
public var title: String {
        didSet {
            mutableNotificationContent?.title = title
        }
}
```